src/utils/managing.officer.corporate.ts

```diff
@@ -129,6 +136,15 @@ const setOfficerData = (reqBody: any, id: string): ApplicationDataType => {
     data[PublicRegisterNameKey] = "";
     data[RegistrationNumberKey] = "";
   }
+
+  // only set start_date and resigned_on keys for Update journey
+  if (reqBody['start_date-day']){
+    data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
+  }
+  if (reqBody['is_still_mo']){
```

Comment:
what if reqBody['is_still_mo'] === 0 - you won't get to line 145. Use contains?

Reply:
will up date to use `'start_date-day' in reqBody` and `'is_still_mo' in reqBody`